setup.py

```diff
@@ -150,15 +150,22 @@ def read_version():
     with (open(os.path.join(TOP_DIR, 'version.txt'), "r")) as f:
         version_str = f.readline().strip()
 
+    # special handling for Onebranch building
+    if os.getenv('BUILD_SOURCEBRANCHNAME', "").startswith('rel-'):
+        return version_str
+
     # is it a dev build or release?
+    rel_br, cid = read_git_refs() if os.path.isdir(
+        os.path.join(TOP_DIR, '.git')) else (True, None)
+
+    if rel_br:
+        return version_str
```

Comment:
So to confirm, in addition to build_id, we are also removing cid from the release branch correct?

Reply:
Yes, if any release branch information retrieved, we will remove the build id or other information from the version string.